recipe/SPQR_TBB.patch

```diff
@@ -0,0 +1,11 @@
+--- SPQR/Lib/Makefile	2016-05-04 16:47:24.000000000 -0400
++++ SPQR/Lib/Makefile.new	2016-05-31 15:04:27.075922172 -0400
+@@ -13,7 +13,7 @@
+ include ../../SuiteSparse_config/SuiteSparse_config.mk
+ 
+ # SPQR depends on CHOLMOD, AMD, COLAMD, LAPACK, the BLAS and SuiteSparse_config
+-LDLIBS += -lamd -lcolamd -lcholmod -lsuitesparseconfig $(LAPACK) $(BLAS)
++LDLIBS += -lamd -lcolamd -lcholmod -lsuitesparseconfig $(LAPACK) $(BLAS) $(TBB)
```

Comment:
Are we sure this patch is needed? It seemed to pick up TBB fine in 4.4.6 and 4.4.5. If this is indeed required, it might be good to drop Tim Davis a line about this regression.